test/feeds.test.js

```diff
@@ -62,3 +62,65 @@ describe('test /feeds/:id responses', () => {
     expect(res.body instanceof Array).toBe(false);
   });
 });
+
+describe('test POST /feeds endpoint', () => {
+  const createdItems = 5;
+  const feedData = {
+    author: 'foo',
+    url: 'http://telescope200.cdot.systems',
+  };
+
+  // Array of feeds
+  const feeds = [...Array(createdItems).keys()].map(item => {
+    return new Feed('foo', `http://telescope${item}.cdot.systems`);
+  });
+
+  beforeAll(() => Promise.all(feeds.map(feed => feed.save())));
+
+  it('responds with json', async () => {
+    const res = await request(app)
+      .post('/feeds')
+      .send(feedData)
+      .set('Accept', 'application/json');
+    expect(res.status).toEqual(201);
+  });
+
+  it('no author being sent', async () => {
+    feedData.author = null;
```

Comment:
Since, to my knowledge, Jest [test methods](https://jestjs.io/docs/en/api#testname-fn-timeout) run in parallel, modifying a shared resource like this might lead to a race condition.

Reply:
That makes more sense why I had to use the 200.